docs/binder_governance.rst

```diff
@@ -0,0 +1,145 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communications, educational materials, and scientific work
+with computational environments where their work can be run and
+shared with others.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
```

Comment:
is the unit the environment or the repository?

Reply:
Environment, I think. GitHub's units are repositories, we use them to build an environment people can run.